internal/httpclient/api_v0alpha1.go

```diff
@@ -1387,6 +1404,132 @@ func (a *V0alpha1ApiService) AdminGetIdentityExecute(r V0alpha1ApiApiAdminGetIde
 	return localVarReturnValue, localVarHTTPResponse, nil
 }
 
+type V0alpha1ApiApiAdminIdentitySessionRequest struct {
+	ctx        context.Context
+	ApiService V0alpha1Api
+	id         string
+}
+
+func (r V0alpha1ApiApiAdminIdentitySessionRequest) Execute() (*SuccessfulAdminIdentitySession, *http.Response, error) {
+	return r.ApiService.AdminIdentitySessionExecute(r)
+}
+
+/*
+ * AdminIdentitySession Calling this endpoint issues session for given identity.
+ * This endpoint is useful for:
+
+Issuing session or session token for a given identity without authenticating
+ * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+ * @param id ID is the identity's ID.
+ * @return V0alpha1ApiApiAdminIdentitySessionRequest
+*/
+func (a *V0alpha1ApiService) AdminIdentitySession(ctx context.Context, id string) V0alpha1ApiApiAdminIdentitySessionRequest {
+	return V0alpha1ApiApiAdminIdentitySessionRequest{
+		ApiService: a,
+		ctx:        ctx,
+		id:         id,
+	}
+}
```

Comment:
Can you explain why we gotta introduce 3 methods here since the two are pretty much invoking the third?

Reply:
Those are automatically generated methods, by documentation tool